### ISSUE TITLE
How (other) players connect is undocumented

### ISSUE THREAD

**Issue by YukiTrace** [open-toontown/open-toontown#101]
I haven't tried to install yet, because I don't know what launcher client software anyone can use to connect and want to know that before proceeding. My understanding is all launchers I can find are designed for and connect to popular servers. As someone who isn't much of a programmer but regularly installs open source stuff I am confused. The best nugget I got on launchers was the original Toontown launcher can be pointed to another domain with a config file. Only I couldn't track down that launcher. Please provide a link to a recommended (an ideally simple) launcher for clients and note OSes it can run on. If there are any unusual networking quirks please provide them.

**Comment by KiraSlith** [comment]
From what I can find, just connecting without building dedicated executables involves just changing the ip address from 127.0.0.1 to your local or remote IP [EDIT: I got this wrong. Apparently there's supposed to be a variable called game-server in configrc.prc, but that's missing in the current commit?]. The real problem is packaging the launcher for clients so they don't have to basically install the entirety of OTTO to just to play. 

I know Panda3D is built around using their version of setuptools, but I can't find a premade setup.py to build with anywhere in the repo, I have no idea what panda3D plugins need to be imported during the build process, I've never worked with Panda3D so I don't know how to pull Panda3D's wheels to build for win_amd64 or linux_amd64, .egg files are scattered everywhere and need to be built down to .bam for clients. The only thing I'm pretty sure about is needing to point setuptools's gui_apps variable to "toontown\launcher\QuickLauncher.py" since it's the one that spawns the GUI.

Short version, I know enough to know I don't know nearly enough, and we need a dev familiar with the project's structure to make it run. [EDIT: I opened a separate issue regarding this.]

**Comment by YukiTrace** [comment]
thanks for your comment. I'd love see setting up a Toontown server be more accessible. I appreciate this project for existing and taking it some of the way there. I hope someone maybe from this group. takes it even further by providing a ready made client or docker image or anything to help make it easier. Taking things from needing a person very advanced user knowledgeable about the game to set up a server, to something a more intermediate user could do would really make it so more friend groups can experience this game. lot's of us have fond memories of this game. I doubt too many people using this are trying to compete with the big servers.

I hope this didn't come across too negative. I appreciate all the work you've done. I just hope accessibility is a priority.

**Comment by YukiTrace** [comment]
sorry didn't mean to close it

**Comment by KiraSlith** [comment]
Ref'd just in case.
